e2e-tests/server/hub.py

```diff
@@ -11,9 +11,9 @@
 class Emitter:
     POLL_TIMEOUT = 3.0
 
-    def __init__(self):
-        self._done = False
-        self._queue = asyncio.Queue()
+    def __init__(self) -> None:
+        self._done: bool = False
+        self._queue: asyncio.Queue[SSEvent] = asyncio.Queue()
```

Comment:
Add `from future import annotations ` otherwise this is unrunnable in python<3.9

Reply:
But does it really need to be run on <3.9?

Reply:
Not really required, but that import may make sense in any case

Reply:
Indeed, added now, and the file compiles under 3.8.